docs/coloring.md

```diff
@@ -37,7 +39,17 @@ To select a customize color:
     -  **Red**, **Green**, **Blue**: Enter the RGB Color. The triangle dial selector will move then to the selected RGB value.
     -  **Opacity**: Enter in the value desired (0 - 255). The lower the number chosen the more translucent the color will be.
     -  **Color Name**: If you know the color name you want you can directly input the value. Entering in a number will change the RGB color and move the dial selector settings to the corresponding color. If the RGB color or the triangle dial selector was used to select a color this color name value will be updated.  
-    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color.  An equivalent "Eye Dropper" type tool can also be accessed from the **Tools** menu (referenced as a Color Picker ![color picker](img/overview/picker.png)). 
+    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  

```

Comment:
>     -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  

I would write:

>     -  **Color Picker**: Select the **Color Picker** ![eye dropper](img/color/eyedropper.png) and then choose a color anywhere on your screen to duplicate that color. The corresponding color values will be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog boxes.

Reply:
No - this is incorrect - there is a difference between the eye dropper and color picker. The eye dropper is in the Choose Primary Color window that pops up when selecting colors and the Color picker is the tool on the tool menu. That is why I partly made a new heading for the Color picker ... you need to see the whole color section to understand this-

Reply:
ahhhhhh i see what you mean now